packages/thriver-core/lib/templates/canvas.html

```diff
@@ -1,4 +1,5 @@
 <template name="canvas">
+    {{>browseHappy}}
```

Comment:
Can we add a space for templates?  So it would look like this:
`{{> browseHappy}}`.  We don't have spacebars linting yet, but we should at least keep some kind of consistency in the meantime.  :)